source/mainnet/net/guides/run-local-chain.rst

```diff
@@ -0,0 +1,185 @@
+    kind = "fresh"
+    balance = "1000000000000000"
+    stake = "500000000000000"
+    template = "baker"
+    identityProvider = 0
+    numKeys = 1
+    threshold = 1
+    repeat = 1 # Changed from 5
+
+.. Note::
+
+    Note that the staked amount needed to participate in finalization committee is some fraction of the total amount of existing CCD, defined by the value of the ``capitalBound`` key in the configurations file. The total amount is the sum of the balances of all the baker and foundation accounts specified in the genesis configuration file. In this particular example the stake is sufficient for baking.
+
+Next, generate the genesis data:
+
+.. code-block:: console
+
+    $ ./target/release/genesis-creator generate --config ./examples/genesis5.toml
+    Deleting any existing directories.
+    Account keys will be generated in ./accounts
+    Chain update keys will be generated in ./update-keys
+    Identity providers will be generated in ./idps
+    Anonymity revokers will be generated in ./ars
+    Baker keys will be generated in ./bakers
+    Cryptographic parameter will be generated in ./global
+    The genesis data will be stored in ./genesis.dat
+    The genesis hash will be written to ./genesis_hash
+    There are 2 accounts in genesis, 1 of which are bakers.
+    Genesis time is set to 2023-05-22 15:08:19.803 UTC.
+    Average block time is set to 5000ms.
+    DONE
+
+In particular, the files of interest to you are ``./genesis.dat`` containing the genesis block data and ``./bakers/baker-0-credentials.json`` containing the credentials of the single baker account that was created.
+
+
+Running the chain
+=================
+
+Now run the chain by starting a single baker node. The node expects the ``genesis.dat`` to reside in a configuration directory, so first create a working directory for the node data and configuration and copy ``genesis.dat`` to it:
+
+.. code-block:: console
+
+    mkdir localchain-node-0
+    cp ./genesis.dat localchain-node-0/
+
+Now run the baker node as follows:
+
+.. code-block:: console
+
+    concordium-node \
+      --no-bootstrap= \
+      --grpc2-listen-addr 127.0.0.1 \
+      --grpc2-listen-port 20001 \
+      --data-dir localchain-node-0 \
+      --config-dir localchain-node-0 \
+      --baker-credentials-file bakers/baker-0-credentials.json
+
+The ``--no-bootstrap`` option lets the node know not to connect to a bootstrapper node for retrieving peers since no peers are in this network so this is not relevant. The ``--grpc2-listen-port`` specifies the port to listen on for Node GRPC V2 API handshakes. We will use this interface to communicate with the node. The ``--data-dir`` and ``--config-dir`` specifies the working directories of the node instance, where its state and configuration is stored. The ``--baker-credentials-file`` instructs the node to run as the baker specified in the supplied credentials file, in this case your generated baker credentials output from the ``genesis-creator`` tool.
```

Comment:
```suggestion
The ``--no-bootstrap`` option lets the node know not to connect to a bootstrapper node for retrieving peers since no peers are in this network so this is not relevant. The ``--grpc2-listen-port`` specifies the port to listen on for Node GRPC V2 API handshakes. You will use this interface to communicate with the node. The ``--data-dir`` and ``--config-dir`` specify the working directories of the node instance, where its state and configuration are stored. The ``--baker-credentials-file`` instructs the node to run as the baker specified in the supplied credentials file, in this case, your generated baker credentials output from the ``genesis-creator`` tool.
```